.scripts/create_push_client.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+function git_configure_and_clone() {
+    git config --global user.name "FABRIC8 CD autobot"
+    git config --global user.email fabric8cd@gmail.com
+
+    set +x
+    echo git clone https://XXXX@github.com/${GHORG}/${GHREPO}.git --depth=1 /tmp/${GHREPO}
+    git clone https://$(echo ${FABRIC8_HUB_TOKEN}|base64 --decode)@github.com/${GHORG}/${GHREPO}.git --depth=1 /tmp/${GHREPO}
+    set -x
+}
+
+function generate_client_and_create_pr() {
+    make docker-generate-client
+
+    local newVersion=${LATEST_COMMIT}
+    local message="chore: update client version to ${newVersion}"
+    local body=$(pr_body)
+    local short_head=$(git rev-parse --short HEAD)
+    local branch="client_update_${short_head}"
+
+    cd /tmp/${GHREPO}
+    git checkout -b ${branch}
+    cd -
+    cp -r cluster tool /tmp/${GHREPO}
+    git rev-parse --short HEAD > /tmp/${GHREPO}/source_commit.txt
+    cd /tmp/${GHREPO}
+
+    git commit cluster tool source_commit.txt -m "${message}"
+    git push -u origin ${branch}
+    rm -rf /tmp/${GHREPO}
+
+    set +x
+    curl -s -X POST -L -H "Authorization: token $(echo ${FABRIC8_HUB_TOKEN}|base64 --decode)" \
+         -d "{\"title\": \"${message}\", \"body\": \"${body}\", \"base\":\"master\", \"head\":\"${branch}\"}" \
```

Comment:
Can we also add a link to the original commit in the main repo to the PR?
I would like to see what actually changed in the main repo which triggered that PR.

Reply:
sure I'll improve it in next pr